packages/atomic/src/components/search-box-suggestions/suggestions-common.ts

```diff
@@ -0,0 +1,41 @@
+import {SearchBox} from '@coveo/headless';
+import {VNode} from '@stencil/core';
+import {buildCustomEvent} from '../../utils/event-utils';
+import {Bindings} from '../../utils/initialization-utils';
+
+export interface SearchBoxSuggestionElement {
+  value: string;
```

Comment:
Unless I am mistaken, we use `value` here as a `key` for JSX code. 

Could we just name it `key` then ?

Reply:
This could change when adding results, but the way suggestion worked is the value (used for key & data-value) will affect the query when you use the arrows, just like on google.com & the JSUI